src/outputter.py

```diff
@@ -0,0 +1,22 @@
+def output_formatter(objects: list, serializer_function):
+    """
+
+    :param objects: A generator of objects
+    :param serializer_function: The serialization function, should be able to serialize a list.
+    :return:
+    """
+
+    return serializer_function(objects)
```

Comment:
Skal vi ligesom ved lokaliseringen lave checks for om det der kommer ind er det vi forventer? F.eks. om "serializer_function" er en funktion?